test/stdio/generator.js

```diff
@@ -526,6 +526,41 @@ const testMultipleYields = async (t, final) => {
 test('Generator can yield "transform" multiple times at different moments', testMultipleYields, false);
 test('Generator can yield "final" multiple times at different moments', testMultipleYields, true);
 
+const partsPerChunk = 4;
+const chunksPerCall = 10;
+const callCount = 5;
+const fullString = '\n'.repeat(getDefaultHighWaterMark(false) / partsPerChunk);
+
+const yieldFullStrings = function * () {
+	yield * Array.from({length: partsPerChunk * chunksPerCall}).fill(fullString);
+};
+
+const manyYieldGenerator = async function * () {
+	for (let index = 0; index < callCount; index += 1) {
+		yield * yieldFullStrings();
+		// eslint-disable-next-line no-await-in-loop
+		await setImmediate();
+	}
+};
```

Comment:
What this is doing: use a transform that calls `yield` multiple times.
It essentially calls `yield '\n\n\n...\n'` 40 times, then repeat this 5 times, i.e. does it 200 times.
Each `yield` contains 4096 bytes.